src/app/courses/course/course.component.ts

```diff
@@ -8,6 +8,7 @@ import { ICourse } from '../shared/course.model';
   styleUrls: ['./course.component.scss'],
 })
 export class CourseComponent implements OnInit {
+  @Input() ngClass: string | string[] | Set<string> | { [klass: string]: any; };
```

Comment:
You don't need it. ngClass is a build it angular directive. The only thing you need to use it on the template level is CommonModule importing.